packages/components/src/border-control/border-control/README.md

```diff
@@ -58,49 +54,49 @@ colors are organized by multiple origins.
 
 Each color may be an object containing a `name` and `color` value.
 
-- Required: No
-- Default: `[]`
+-   Required: No
+-   Default: `[]`
 
 ### `disableCustomColors`: `boolean`
 
 This toggles the ability to choose custom colors.
 
-- Required: No
+-   Required: No
 
 ### `disableUnits`: `boolean`
 
 This controls whether unit selection should be disabled.
 
-- Required: No
+-   Required: No
 
 ### `enableAlpha`: `boolean`
 
 This controls whether the alpha channel will be offered when selecting
 custom colors.
 
-- Required: No
-- Default: `false`
+-   Required: No
+-   Default: `false`
```

Comment:
If the above is correct and `enableAlpha` defaults to `true` intentionally, shouldn't we invert the `WithAlphaEnabled` story to demonstrate a "WithAlphaDisabled" behavior instead?

Reply:
Makes sense, done.

Reply:
On second thought, is this story really that helpful? Maybe it is in the underlying component (`ColorPalette`, which btw doesn't have it), but here it seems pretty useless when the control is right there. We might be able to get away with just removing it.

Reply:
Yeah I think I'm just going to remove it. Doesn't seem useful vs just using controls.